dashboard/app/views/levels/_level_group.haml

```diff
@@ -39,21 +28,19 @@
       - last_attempt = ""
     - level_class = level.class.to_s.underscore
 
-    - level.properties['title'] = nil
-
     - if level_class == "multi"
       -# For students (or teachers doing PD), mark all answers incorrect, so we don't give anything away.
       - unless current_user.try(:authorized_teacher?) && !@script.try(:pd?)
         - level.properties['answers'].each { |answer| answer['correct'] = false }
-      = render partial: 'levels/single_multi', locals: {page: page, index: index, number: prior_page_count + index + 1, standalone: false, level: level, last_attempt: last_attempt, tight_layout: true }
-    - elsif level_class = "text_match"
```

Comment:
I guess it was the fall-through case by the time it got to here, so it did.  Fixed, regardless!